### PR TITLE
virt-edge-removed-temp

### PR DESCRIPTION
**Pull request by resmaa**
Removed edge-virtualized page temporarily.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1016#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge.md [hunk 1]
```diff
@@ -51,6 +51,14 @@ Palette Edge Platform provides two solutions for edge, designed to support large
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:51]: @resmaa let's remove the beta tag

CleanShot 2022-12-02 at 08 41 24

File: content/docs/04-clusters/03-edge.md [hunk 2]
```diff
@@ -51,6 +51,14 @@ Palette Edge Platform provides two solutions for edge, designed to support large
...
+<WarningBox>
+
+Palette recommends Virtualized Edge deployment only when the user has a **single edge appliance and needs HA virtualized Kubernetes Cluster.**
+
+[Contact Spectro support via the Service Desk](http://support.spectrocloud.io/) for more details on the deployment of Virtualized Edge Architecture.
+
+</WarningBox>
+
 
 ### Native Edge
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:63]: Let's put the Native Edge section first, so that's what practitioners see first.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/03-edge.md [hunk 3]
```diff
@@ -47,17 +47,25 @@ Palette Edge Platform provides two solutions for edge, designed to support large
...
 
 Designed for sites that typically have a single large bare-metal appliance. Virtualized nodes are instantiated on the appliance using libvirt, and the desired version of OS and Kubernetes is deployed on the nodes. Each Virtual Machine (VM) represents a Kubernetes node. Users can specify placement settings for these virtual machines to ensure they are launched in the desired network and storage pools. Users can also configure VM hardware settings such as CPU, Memory, Disk size, etc.
 
+<br />
 
-### Native Edge
+<WarningBox>
 
-Designed for sites that typically have one or more small form factor appliances such as intel NUC. An instance of Palette Edge Distribution based on the desired version of OS and K8s is installed natively onto the devices. Palette Management Console manages the day-1 installation as well as all the day-2 activities, such as scaling, upgrades, reconfiguration, etc.
+Palette recommends Virtualized Edge deployment only when the user has a **single edge appliance and needs HA virtualized Kubernetes Cluster.**
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:64]: This should not be in bold as it goes against our style guide.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1016#pullrequestreview-1202717343" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1016#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**